docs/source/01_introduction/01_tutorial.md

```diff
@@ -0,0 +1,131 @@
+# Tutorial
+
+
+## Introduction
+
+## Pre-requisite
+
+### Create a virtual environment
+
+It is recommended to create a virutal environment to install your packages. You can install ``kedro-pandera`` from PyPI.
+```bash
+conda create -n kedro-pandera-tutorial python=3.10 -y
+conda activate kedro-pandera-tutorial
+pip install kedro-pandera
+```
+
+Notice that installing ``kedro-pandera`` will automatically install ``kedro``.
+
+```{note}
+``kedro-pandera`` only works with ``kedro>=0.18.13``. You may need to upgrade your kedro version if you are using it in an older project.
```

Comment:
I was wondering this too. Honestly  I don't care to commit to retrocompatiblity too soon. I think we could add support for ``TemplatedConfigLoader`` later if it is a strongly required feature and once the design is mature. For my personal experience, apart from the very first versions of the 0.18.X series, you can just upgrade an existing project and everything works fine. I guess people with say >=0.18.3 will be able to upgrade to 0.18.13 and use the plugin with no effort. 

``OmegaConfigLoader`` custom resolvers offer a very neat solution to several problems I had during development (anticipating support for different format of DataFrameSchema like python or yaml, having a very simple syntax to load the schema, and having the schema object directly generated form config without a lot of boilerplate custom loading code inside the hook) and I though the advantages were worth the low retrocompatiblity but it can be challenged later. I guess the goal is to have a mature design for the 0.19.X series, and people will move toward the plugin progressively.  

I'll describe in detail in the PR what I did to have you review before we merge. My sort term goal is rather to find out a way to ensure a smooth developer experience and implement the most desired feature.

Reply:
I'm a fan of not supporting TemplatedConfigLoader - this is a good carrot to get people to update!

Reply:
Not sure if you're non partisan here @datajoely but I won't complain since we agree, sorry @noklam ;)
TBH I can't be involved as much as I can (there are 15 opened issues on ``kedro-mlflow`` while I am writing this, kedro-serving is in a stale for 2 years, and I've created the repo for kedro pandera for serveal months and never had time to develop before I was in holidays) so let's not be too ambitious for now. We can obviously refactor it later.